dashboard/src/components/UpgradeForm/UpgradeForm.test.tsx

```diff
@@ -350,21 +346,23 @@ describe("when receiving new props", () => {
     bar1: value1
 `,
       deployedValues: `foo:
-  - foo1: 
+  - foo1:
     bar1: value1
-  - foo2: 
+  - foo2:
     bar2: value2
 `,
       newDefaultValues: `foo:
     - foo1:
       bar1: value1
 `,
-      result: `foo:
-  - foo1: 
-    bar1: value1
-  - foo2: 
-    bar2: value2
-`,
+      result: [
+        `foo:`,
+        `  - foo1: `,
```

Comment:
Apparently the space is in the results that we're comparing, but our (or my) editor removes trailing spaces if I format it as a string as previously. This just allows including the trailing spaces (could also just do as a normal string "foo: \n  - foo1: \n...")

Reply:
I just ended up temporarily disabling this vscode feature to save this test :P, but your solution is much better